src/reanimated2/frameCallback/FrameCallbackRegistryUI.ts

```diff
@@ -14,21 +29,44 @@ export const prepareUIRegistry = runOnUI(() => {
   'worklet';
 
   const frameCallbackRegistry: FrameCallbackRegistryUI = {
-    frameCallbackRegistry: new Map<number, () => void>(),
+    frameCallbackRegistry: new Map<number, CallbackDetails>(),
     frameCallbackActive: new Set<number>(),
     isFrameCallbackRunning: false,
+    previousFrameTimestamp: undefined,
 
     runCallbacks() {
-      const loop = () => {
+      const loop = (timestamp: number) => {
+        if (this.previousFrameTimestamp === undefined) {
+          this.previousFrameTimestamp = timestamp;
+        }
+
+        const timeSinceLastFrame = timestamp - this.previousFrameTimestamp;
```

Comment:
We just assigned `this.previousFrameTimestamp = timestamp;` so effectively this will be `timestamp - timestamp`, right?

Reply:
Yes, but only on the first frame. Do you think there is a better way of doing this?